events/janus_mqttevh.c

```diff
@@ -633,6 +663,41 @@ static int janus_mqttevh_init(const char *config_path) {
 	qos_item = janus_config_get(config, config_general, janus_config_type_item, "qos");
 	ctx->publish.qos = (qos_item && qos_item->value) ? atoi(qos_item->value) : 1;
 
+	connect_status_item = janus_config_get(config, config_general, janus_config_type_item, "connect_status");
+	if(connect_status_item && connect_status_item->value) {
+		ctx->publish.connect_status = g_strdup(connect_status_item->value);
+	} else {
+		ctx->publish.connect_status = g_strdup(DEFAULT_CONNECT_STATUS);
+	}
+
+	disconnect_status_item = janus_config_get(config, config_general, janus_config_type_item, "disconnect_status");
+	if(disconnect_status_item && disconnect_status_item->value) {
+		ctx->publish.disconnect_status = g_strdup(disconnect_status_item->value);
+	} else {
+		ctx->publish.disconnect_status = g_strdup(DEFAULT_DISCONNECT_STATUS);
+	}
```

Comment:
But does this mean that the connect/disconnect stuff is ALWAYS sent, now? I'm not sure I'm ok with this: as the last will, this should be configurable and optional. The Janus core already sends a message when starting up, and one when shutting down.

Reply:
Connection stuff was there before. Disconnection stuff was added by me to keep things simmetrical - consider the case where Janus gracefully terminates and retained status message still indicates that Janus is online which is not true.

There is no way to retain such events coming from Janus Core, I guess.

Reply:
Got it, thanks for the clarification.